live-examples/html-examples/content-sectioning/h1-h6.html

```diff
@@ -0,0 +1,12 @@
+<h1 id="title-heading">Page Title (Heading 1)</h1>
```

Comment:
This title wraps on Chrome at a browser width of 1000px, Please make it shorter or reduce the font size.